pkg/cmd/experimental/buildchain/buildchain.go

```diff
@@ -94,12 +93,29 @@ func (o *BuildChainOptions) Complete(f *clientcmd.Factory, cmd *cobra.Command, a
 	}
 	o.c, o.t = oc, oc
 
-	// Parse user input
-	o.name, o.tag, err = buildChainInput(args[0])
-	if err != nil {
-		return cmdutil.UsageError(cmd, err.Error())
+	parts := strings.Split(args[0], "/")
```

Comment:
Ah, build chain uses resource syntax.

Still, the duplication of this code seems high.  Why is split handled outside of resource.Builder?